pkgs/top-level/all-packages.nix

```diff
@@ -42304,4 +42304,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  girouette = callPackage ../applications/misc/girouette { };
```

Comment:
Nit: Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file

Reply:
They'll migrate them manually anyways for what it's worth

Reply:
oh, wasn't aware of the new structure  
I've moved it there